bin/generate-feature-config.php

```diff
@@ -7,7 +7,7 @@
 
 $phase = isset( $_SERVER['WC_ADMIN_PHASE'] ) ? $_SERVER['WC_ADMIN_PHASE'] : ''; // WPCS: sanitization ok.
 if ( ! in_array( $phase, array( 'development', 'plugin', 'core' ), true ) ) {
-	$phase = 'core';
+	$phase = 'development';
```

Comment:
It looks `WC_ADMIN_PHASE` isn't actually passed down to this file when we run `npm run build` (which might also be why you didn't see the message, @rrennick) – we could either change this back to `core`, or change the `build:feature-config` to be `WC_ADMIN_PHASE=core php bin/generate-feature-config.php` … I don't really have a preference either way.

Reply:
@ryelle Thanks again. I've updated the PR & fixed the merge conflicts.